slugger.cabal

```diff
@@ -0,0 +1,63 @@
+cabal-version:        3.0
+name:                 slugger
+version:              0.1.0.0
+synopsis:             Clean URI slugs for Haskell
+description:          Clean URI slugs for Haskell
```

Comment:
It's useful to point to `README.md` here.

Reply:
Is there anything special with that – do you mean a link, or `View README.md`, or nothing more than `README.md`?

Reply:
Just "View README.md" should be enough IMO, since you have the `README.md` on `extra-source-files` it will already be on the Hackage page.